hail/python/hail/matrixtable.py

```diff
@@ -2533,6 +2533,90 @@ def _localize_entries(self, entries_field_name, cols_field_name) -> 'Table':
         return Table(CastMatrixToTable(
             self._mir, entries_field_name, cols_field_name))
 
+    @typecheck_method(entries_array_field_name=nullable(str),
+                      columns_array_field_name=nullable(str))
+    def localize_entries(self,
+                         entries_array_field_name=None,
+                         columns_array_field_name=None) -> 'Table':
+        """Convert the matrix table to a table with entries localized as an array of structs.
+
+        Examples
+        --------
+        Build a numpy ndarray from a small :class:`.MatrixTable`:
+
+        >>> mt = hl.utils.range_matrix_table(3,3)
+        >>> mt = mt.select_entries(x = mt.row_idx * mt.col_idx)
+        >>> mt.x.show()
+        +---------+---------+-------+
+        | row_idx | col_idx |     x |
+        +---------+---------+-------+
+        |   int32 |   int32 | int32 |
+        +---------+---------+-------+
+        |       0 |       0 |     0 |
+        |       0 |       1 |     0 |
+        |       0 |       2 |     0 |
+        |       1 |       0 |     0 |
+        |       1 |       1 |     1 |
+        |       1 |       2 |     2 |
+        |       2 |       0 |     0 |
+        |       2 |       1 |     2 |
+        |       2 |       2 |     4 |
+        +---------+---------+-------+
+
+        >>> t = mt.localize_entries('entry_structs', 'columns')
+        >>> t = t.select(entries = t.entry_structs.map(lambda entry: entry.x))
+        >>> t.describe()
+        ----------------------------------------
+        Global fields:
+            'columns': array<struct {
+                col_idx: int32
+            }> 
+        ----------------------------------------
+        Row fields:
+            'row_idx': int32 
+            'entries': array<int32> 
+        ----------------------------------------
+        Key: ['row_idx']
+        ----------------------------------------
+
+        >>> np.array(t.entries.collect())
+        array([[0, 0, 0],
+               [0, 1, 2],
+               [0, 2, 4]])
+
+        Warning
+        -------
+        This operation may increase the size of a partition. Use with care on
+        a dataset with long rows and large partitions.
+
+        Parameters
+        ----------
+        entries_array_field_name : :obj:`str`
+            The name of the table field containing the array of entry structs
```

Comment:
oh, you added it in the parameter description. Can we move the last sentence of both of these up?